ocaml/quicktest/quicktest_static_vdis.ml

```diff
@@ -0,0 +1,33 @@
+
+let run_cmd cmd =
+  print_endline cmd;
+  Alcotest.(check int) cmd 0 (Sys.command cmd)
+
+let static_vdis args =
+  let static_vdis = "/opt/xensource/bin/static-vdis" in
+  let cmd = static_vdis ^ " " ^ (String.concat " " args) in
+  run_cmd cmd
+
+let test_static_vdis rpc session_id sr_info () =
+  let sr = sr_info.Qt.sr in
+  Qt.VDI.with_new rpc session_id sr (fun self ->
+      let uuid = Client.Client.VDI.get_uuid ~rpc ~session_id ~self in
+      static_vdis ["add"; uuid; "quicktest_static_vdis"];
+      static_vdis ["list"];
+      static_vdis ["attach"; uuid];
+      static_vdis ["detach"; uuid];
```

Comment:
`List.iter static_vdis [ ...]`?

Reply:
In my opinion the code is more readable this way, because it looks like a command :)